pkg/ip/link.go

```diff
@@ -39,37 +39,61 @@ func makeVeth(name, peer string, mtu int) (netlink.Link, error) {
 	return veth, nil
 }
 
+func makeVeth(name string, mtu int) (peerName string, veth netlink.Link, err error) {
+	for {
```

Comment:
Trust very small probabilities not occurring. But not in my coding skills so I'll add a sanity check.